data/upgrades/crew.json

```diff
@@ -952,7 +952,7 @@
       }
     ],
     "cost": {
-      "value": 0
+      "value": 8
```

Comment:
According to https://images-cdn.fantasyflightgames.com/filer_public/ad/23/ad238fa5-3f68-4867-a2a8-8327655eb5c5/onlinepoints_rebelalliance3.pdf Lando is actually 5 points, not 8.

Reply:
This is Scum Lando, not Rebel Lando, he's definitely 8 - https://images-cdn.fantasyflightgames.com/filer_public/08/29/08299173-c31b-4c48-93aa-c9901621adae/onlinepoints_scumandvillainy.pdf

Reply:
D'oh 😅